lib/backup_restore/backup_store.rb

```diff
@@ -10,10 +10,10 @@ class BackupStore
     def self.create(opts = {})
       case opts[:location] || SiteSetting.backup_location
       when BackupLocationSiteSetting::LOCAL
-        require_dependency "backup_restore/local_backup_store"
+        require "backup_restore/local_backup_store"
```

Comment:
I guess this links to https://github.com/discourse/discourse/pull/15649/files#r833871445. In the past, we do not autoload this file and only require it when needed. Right now, the entire `lib` folder is autoloaded so we should not need to manually `require` it here.

Reply:
Same answer as previously, it depends where and when this file is required, if it’s before the app has finished to boot up then we need a require.

Reply:
I think we did this previously because the file isn't supposed to be autoloaded and only required when used. I think we'll resolve this as part of the `app/lib` and `lib` discussion. So resolving this.